meta-openpli/conf/distro/reporefs.conf

```diff
@@ -211,8 +211,8 @@ SRCREV_pn-enigma2-plugin-picons-ziggo.metal-stamp-100x60 = "${AUTOREV}"
 SRCREV_pn-rtl8188eu = "${AUTOREV}"
 SRCREV_pn-rtl8192eu = "${AUTOREV}"
 
-SRCREV_pn-enigma2-plugin-softcams-oscam ?= "${AUTOREV}"
-SRCREV_pn-enigma2-plugin-softcams-oscam-emu ?= "11400"
+SRCREV_pn-enigma2-plugin-softcams-oscam = "5e695a59383af46fce24309040f40bfbb6dde127"
```

Comment:
What's wrong with AUTOREV when it's on our git?

Reply:
To match with svn subversion..If we use AUTOREV we have to drop svn revision.
So e.g. oscam-1.20-git130+afda92b-r0
Or we need to find a way to grab 'svn revision' from GIT if possible?

Reply:
Can you grab it from releases page: https://github.com/PLi-metas/oscam/releases ?

Reply:
Are you using the same mirror tools as oscam.cc used to use? If so, I have added some updates (commited by theparasol) to get the SVN number from the git-svn-id in the commit message (see https://github.com/PLi-metas/oscam/commit/d3b3898cee0b1f4065c689b9b9a4c16d034ce660) so AUTOREV should work fine).